source/versioned-api/tests/transaction-handling.json

```diff
@@ -227,6 +227,162 @@
           ]
         }
       ]
+    },
+    {
+      "description": "Committing a transaction twice does not append server API options",
```

Comment:
Instead of adding a new test case, I could have just added another `commitTransaction` operation to the end of the previous test case. To be exhaustive/descriptive, I've added another case, but let me know if it's overkill.

Reply:
I like it - this way the test name makes it clear that the duplicate `commit` command is not an oversight 👍